CMIP6_studies/Ouranos.yaml

```diff
@@ -0,0 +1,355 @@
+- key: Ouranos tas bias
+  doi: [in-house performance]
+  metric: &tasclim
+    name: tasclim
+    long_name: tas climatology vs ERA5 climatology
+    units: K
+    variables: tas
+    comment: >
+      We calculate the RMSE at each grid point over the selected the domain (relative to ERA5) for the period from 1981 to 2010 and then we do a spatial average.
+      All models and ERA5 are linearly interpolated on a regular 1°x1° grid.  
+      -----
+      Periods, domains can be changed in https://pavics.ouranos.ca/selection-de-modeles
+    best: 1.21
+    worst: 2.26
+  type: performance
+  spatial_scope: NAM-CORDEX
+  temporal_scope: Climatology
+  period:
+    reference: 1981-2010
+  plausible_values:
+    min: 0
+    max: +inf 
+    source: Ouranos
+    comment: > 
+      
+  data_source: author
+  data:
+    EC-Earth3-Veg: 1.21
```

Comment:
data looks good, except for the missing variant_id (i.e. member). This should be e.g. `EC-Earth3-Veg_r1i1p1f1: 1.21`